modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds Fetchpriority parameter to the none lazy loaded images.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since 1.5.0
+ */
+
+/**
+ * Filter image tags in content to add fetchpriority to the image tag if not lazy loaded set.
+ *
+ * @since 1.5.0
+ *
+ * @param string $filtered_image The filtered image.
+ * @param string $context        The context of the image.
+ *
+ * @return string image tag
+ */
+function fetchpriority_img_tag_add( $filtered_image, $context ) {
+
+	// Only apply this in `the_content` for now, since otherwise it can result in duplicate runs due to a problem with full site editing logic.
```

Comment:
interesting, any reference link for this issue?

Reply:
no removed comment